dev-lang/ispc/metadata.xml

```diff
@@ -9,6 +9,9 @@
 		<remote-id type="github">ispc/ispc</remote-id>
 	</upstream>
 	<use>
+		<flag name="cross">
+			Build ISPC with cross target support.
```

Comment:
What does it do? Why would people enable it? Is there a reason not to enable it unconditionally?

Reply:
On Linux it adds support for building for FreeBSD, Android and a custom_linux os whatever that means, I'd rather leave it out then enable it unconditionally.